Gruntfile.js

```diff
@@ -87,12 +87,16 @@ module.exports = function(grunt) {
         bgShell: {
             ember: {
                 cmd: 'npm run build -- --watch',
-                bg: true,
-                stdout: function (out) {
-                    grunt.log.writeln(chalk.cyan('Ember-cli::') + out);
-                },
-                stderror: function (error) {
-                    grunt.log.error(chalk.red('Ember-cli::' + error));
+                bg: true
+            }
+        },
+
+        watch: {
+            csscomb: {
+                files: ['app/styles/**/*.css'],
+                tasks: ['shell:csscombfix'],
+                options: {
+                    livereload: 35730
```

Comment:
Will the livereload do anything in this case? We'd need to inject the live reload script into the admin HTML and set it up to listen to the correct port, maybe setting `livereload: false` will be better until we can utilise it?